lib/pbench/server/auth/keycloak_admin.py

```diff
@@ -0,0 +1,242 @@
+from typing import Dict, List
+
+from pbench.server.auth.auth_provider_urls import (
+    URL_ADMIN_CLIENT,
+    URL_ADMIN_CLIENT_ALL_SESSIONS,
+    URL_ADMIN_CLIENT_ROLE_MEMBERS,
+    URL_ADMIN_CLIENT_ROLES,
+    URL_ADMIN_GET_SESSIONS,
+    URL_ADMIN_KEYS,
+    URL_ADMIN_REALM_ROLES,
+    URL_ADMIN_REALM_ROLES_MEMBERS,
+    URL_ADMIN_USER,
+    URL_ADMIN_USER_CLIENT_ROLES,
+    URL_ADMIN_USER_REALM_ROLES,
+    URL_ADMIN_USERS,
+)
+from pbench.server.auth.keycloak_openid import KeycloakOpenID
+
+
+class Admin(KeycloakOpenID):
+    def __int__(
+        self,
+        server_url: str,
+        realm_name: str,
+        client_id: str,
+        logger,
+        client_secret_key: str = None,
+        verify: bool = True,
+        headers: Dict = None,
+        timeout: int = 60,
+        user_realm: str = None,
+    ):
+        super().__int__(
+            server_url,
+            realm_name,
+            client_id,
+            logger,
+            client_secret_key,
+            verify,
+            headers,
+            timeout,
+        )
```

Comment:
@npalaska, the LGTM bot would like you to know that you've misspelled `__init__()` at lines 21 and 33 -- you've lost the second `i`.